api/apps/api/src/modules/published-project/published-project-crud.service.ts

```diff
@@ -48,15 +48,16 @@ export class PublishedProjectCrudService extends AppBaseService<
     fetchSpecification: FetchSpecification,
     info?: ProjectsRequest,
   ): Promise<SelectQueryBuilder<PublishedProject>> {
-    let showUnderModerationPublishedProjects = false;
     const id = info?.authenticatedUser?.id;
-    if (id && (await this.usersService.isPlatformAdmin(id))) {
-      showUnderModerationPublishedProjects = true;
-    }
 
-    query.andWhere('published_project.underModeration = :underModeration', {
-      underModeration: showUnderModerationPublishedProjects,
-    });
+    /*
+      If we are listing projects for non-authenticated requests or for
+      authenticated users who are not admin, projects under moderation
+      will be hiding from the listing.
+    */
+    if (!id || !(await this.usersService.isPlatformAdmin(id))) {
```

Comment:
while we are revisiting this, let's make the name more explicit
```suggestion
    if (!userId || !(await this.usersService.isPlatformAdmin(userId))) {
```